Pipfile

```diff
@@ -21,4 +21,4 @@ yamllint = "1.26.3"
 managedtenants_cli = {editable = true,path = "."}
 
 [requires]
-python_version = "3.9"
+python_version = "3"
```

Comment:
Currently there doesn't seem to be a way to provide a range of verisons like `">=3.6"`. Setting it to `"3"` will make sure the it works on all python versions, although we should try and avoid verisons lower than `"3.6"`